baseplate/thrift/baseplate.thrift

```diff
@@ -71,4 +71,5 @@ struct Request {
     1: Loid loid;
     2: Session session;
     3: AuthenticationToken authentication_token;
+    4: string country_code;
```

Comment:
It might make sense to wrap this in a struct for grouping this with other similar data?

Reply:
Sounds reasonable, how does this sound?
```
struct Client {
    1: string country_code;
}
```

Reply:
Isn't the entire edge context about the client?

Reply:
That's true, based on that I actually don't think we need a struct for this

Reply:
Sgtm

Reply:
It could be a GeoLocation struct or something like that so that if we changed from ISO country code to some other format down the line we'd have a place to put the multiple fields.

Reply:
A custom type alias might be nice too. Just to make the value of the field super obvious.